controllers/userController.js

```diff
@@ -0,0 +1,108 @@
+const userService = require("../services/dbUserServices");
+
+class UserController {
+  async getAllUsers(req, res) {
+    try {
+      const users = await userService.getAllUsers();
+      if (!users) {
+        res.status(200).json({
+          status: "error",
+          message: "user didn't created yet",
+        });
+      }
+      res.status(200).json({
+        users,
+        status: "success",
+      });
+    } catch (e) {
+      res.status(500).json({
+        message: "unexpected server error",
+      });
+    }
+  }
+
+  async addNewUser(req, res) {
+    try {
+      const isUserCreated = await userService.addNewUser(req.body);
+      if (!isUserCreated) {
+        res.status(200).json({
+          status: "error",
+          message: "user has not been created",
+        });
+      }
+      res.status(200).json({
```

Comment:
There is a good practice to return created entity or its id in response

Reply:
yes, it's good